### PR TITLE
[FEATURE BUGFIX] Added Theme features and solved multiple bugs

### PR DESCRIPTION
**Pull request by laaledesiempre**
Solved multiple bugs and added
New theme saving system
Theme saves with background
Solved missing background not hidding wrapper
Added sounds menu
added github pages workflow
Added new building scripts
Added serving dependencies for future serving stuff

### CODE REVIEW

**Comment by Dpeta** [comment]
Maybe we're jumping ahead a bit here, I'm not using pages so idnk if upstream needs the workflow and extra dependencies for that.

**Comment by laaledesiempre** [comment]
removed all that studd and updated the advanced mode styles (still have things to do with the check box, but now it looks the same as the regular login)

**Comment by Dpeta** [comment]
yippee

File: style.css [hunk 1]
```diff
@@ -33,7 +34,8 @@
   font-family: Courier;
   font-weight: 1000;
   font-size: 1rem;
-  color: var(--black)
+  color: var(--black);
+  text-shadow:0px 0px 2px var(--white);
```
Comment (Dpeta) [inline style.css:38]:
```suggestion

```
Maybe we could do without text shadow? White shadow on white texts gives a kinda glowy/blurry effect for the chum/memo lists which idnk if is desirable.

image
image

image
image

don't mind memo/chums tab buttons though with the darker text & background
image
image

**Pull request merged**